packages/fetch-cached-dns/package.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "@vercel/fetch-cached-dns",
+  "version": "2.0.2",
+  "description": "A decorator on top of `fetch` that caches the DNS query of the `hostname` of the passed URL",
+  "license": "MIT",
+  "files": [
+    "index.js",
+    "index.d.ts",
+    "util.js"
+  ],
+  "scripts": {
+    "test": "jest test",
+    "lint": "eslint .",
+    "lint:staged": "lint-staged"
+  },
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/vercel/fetch-cached-dns.git"
+  },
```

```suggestion
  "repository": {
    "type": "git",
    "url": "https://github.com/vercel/fetch.git",
    "directory": "packages/fetch-cached-dns"
  },
```

This should include the directory